article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -13,7 +13,7 @@
         @* Add advert every 5 blog posts, up to a maximum of 8 *@
         @if(blockGroup.length == 5 && index < 8) {
             <div id="amp-ad-@index" data-sort-time="1" class="block amp-ad-container amp-ad-container--live-blog">
```

Comment:
@NataliaLKB how about this? 

https://github.com/guardian/frontend/pull/15580/files#diff-0bc1c927ebbe0f3e69d8b1ccbc7efef7R93